libnmstate/nm/applier.py

```diff
@@ -359,10 +361,20 @@ def _build_connection_profile(iface_desired_state, base_con_profile=None):
     if base_profile:
         con_setting.import_by_profile(base_con_profile)
     else:
+        # The OVS interface will refer this port via connection.uuid
+        # The NM does not allows empty interface name, set to _ which is not
```

Comment:
Does the code really do what is written in the comment?

Reply:
No. That is left over. The `_` does not works.